src/containers/Category.js

```diff
@@ -232,6 +232,30 @@ function postsProp(state, listId, page) {
   return posts
 }
 
+/**
+ *  @param {string} categoryPath - category pathname
+ *  @param {string} subcategoryPath - subcategory pathname
+ *  @return {Object} - subcategoryList, activeTabIndex
+ */
+function titleTabProp(categoryPath, subcategoryPath) {
+  const subcategoryList = _.map(CATEGORY_SET[categoryPath], subcategoryKey => {
+    const path = subcategoryKey
+    const label = SUBCATEGORY_LABEL[subcategoryKey]
+    const id = SUBCATEGORY_ID[subcategoryKey]
+    const link =
+      subcategoryKey !== 'all'
+        ? `/categories/${categoryPath}/${path}`
+        : `/categories/${categoryPath}`
+    const isExternal = false
+    return { id, label, link, path, isExternal }
```

Comment:
`isExternal` is a constant & we only use once, so probably we could just replace this variable with `false` & use comment to show it's a boolean meaning `isExternal`

Reply:
updated ([ac2670c](https://github.com/twreporter/twreporter-react/pull/2343/commits/ac2670c5c5e4145142df2d4b7a2c6fbbb804121c))